ui/app/templates/components/auth-info.hbs

```diff
@@ -30,27 +31,23 @@
               </button>
             </li>
             <li class="action">
-              <ConfirmAction
-                @buttonClasses="button link is-destroy"
-                @confirmTitle={{concat "Revoke " (get auth 'authData.displayName') "?"}}
-                @confirmMessage={{concat "You will not be able to log in again with this token."}}
-                @confirmButtonText="Revoke"
-                @onConfirmAction={{action "revokeToken"}}
-              >
-                Revoke token
-              </ConfirmAction>
+              <c.Trigger
+                @id={{get auth 'authData.displayName'}}
+                @title={{concat "Revoke " (get auth 'authData.displayName') "?"}}
+                @onConfirm={{action "revokeToken"}}
+                @message="You will not be able to log in again with this token."
+                @onTrigger={{action c.onTrigger (get auth 'authData.displayName')}}
```

Comment:
Does `onTrigger` need to be exposed? I think the trigger already has access to the confirm state and the id, which is what this is firing. Is there ever a case where we'd want to do something other than just showing the message?

Reply:
this is a great question, thank you for bringing it up! right now i can't think of or find an existing use case where the trigger should do anything other than show a confirmation message. i think a more realistic scenario is where the developer would want something custom to happen after deleting a thing (perhaps fetching some data or kicking off a series of tasks), and that scenario should be covered by passing in a custom function to `onConfirm`. 

so for now i don't think we need to expose `onTrigger`. i've updated the component to reflect that.